seed/ts-sdk/basic-auth-environment-variables/src/api/resources/basicAuth/client/Client.ts

```diff
@@ -198,9 +198,23 @@ export class BasicAuth {
     }
 
     protected async _getAuthorizationHeader(): Promise<string | undefined> {
+        const username = (await core.Supplier.get(this._options.username)) ?? process?.env["USERNAME"];
+        if (username == null) {
+            throw new errors.SeedBasicAuthEnvironmentVariablesError({
+                message: "Please specify USERNAME when instantiating the client.",
+            });
+        }
+
+        const accessToken = (await core.Supplier.get(this._options.accessToken)) ?? process?.env["PASSWORD"];
+        if (accessToken == null) {
+            throw new errors.SeedBasicAuthEnvironmentVariablesError({
+                message: "Please specify PASSWORD when instantiating the client.",
```

Comment:
nit: `Please specify a password by either passing it in to the constructor or initializing a PASSWORD environment variable`

Something like this ^ would be a better error message i think

Reply:
Sweet, I just pattern matched on the original message we used for bearer. We can update across the board.